modules/prediction/proto/lane_graph.proto

```diff
@@ -60,6 +60,7 @@ message LaneSequence {
   optional double lane_s = 17;
   optional double lane_l = 18;
   optional StopSign stop_sign = 20;
+  optional int32 right_of_way = 21 [default = 50];
```

Comment:
1. Change to unsigned int if it's supposed to be.
2. Comment about the supposed range.